lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -307,13 +309,24 @@ public string GetSecurityTokenIdentifierFileName()
         /// The filename for security token password
         /// </summary>
         /// <returns>filename which contains raw bytes of security token password</returns>
+        /// TODO: [JIRA REEF-1887] Will remove.
         [Unstable("0.13", "Security token should be handled by .NET only REEF client in the future")]
         public string GetSecurityTokenPasswordFileName()
         {
             return SECURITY_TOKEN_PASSWORD_FILE;
         }
 
         /// <summary>
+        /// File name for security token information.
+        /// It supersedes GetSecurityTokenPasswordFileName() and GetSecurityTokenIdentifierFileName().
```

Comment:
Mark this line with TODO so we can remove it later, e.g. 
```C#
// TODO[JIRA REEF-1887] supersedes GetSecurityTokenPasswordFileName() and GetSecurityTokenIdentifierFileName(). Remove this comment line in 0.17
```

Reply:
done